app/src/main/java/com/deframe/artapp/ui/CollectionActivity.kt

```diff
@@ -0,0 +1,106 @@
+package com.deframe.artapp.ui
+
+import android.support.v7.app.AppCompatActivity
+import android.os.Bundle
+import com.deframe.artapp.R
+import com.squareup.picasso.Picasso
+import kotlinx.android.synthetic.main.activity_collection.*
+import kotlinx.android.synthetic.main.activity_museum_details.*
+import org.json.JSONArray
+import org.json.JSONObject
+import java.net.URL
+import java.util.*
+
+class CollectionActivity : AppCompatActivity() {
+    var collectionArr : JSONArray = JSONArray()
+    var paintingList : LinkedList<JSONObject> = LinkedList()
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_collection)
+
+        val intent = intent
+        var collectionName = intent.getStringExtra("collection")
+        var museumId =  intent.getStringExtra("id")
+
+        var t = Thread(Runnable {
+            var url = "http://deframe-test-api.us-east-1.elasticbeanstalk.com:80/museums/" + museumId + "/gallery"
```

Comment:
Try using ``const`` or ``val``.  We should declare and keep the constants in different place, as this may be used through out the project.

Kindly follow this read up -
https://blog.egorand.me/where-do-i-put-my-constants-in-kotlin/

recommended way is- 
1. Create a util class
class Constants {
  companion object {
   const val API_TEST = "http://deframe-test-api.us-east-1.elasticbeanstalk.com:80"
  }
}

2. Using in the place we need 
```
fun main(args: Array<String>) {
  println(Constants.API_TEST)
}
```